spec/javascripts/analytics_toolkit/govuk-tracker.spec.js

```diff
@@ -52,12 +52,23 @@ describe('GOVUK.GOVUKTracker', function () {
     })
 
     it('adds performance data', function () {
+      if (!window.performance) {
```

Comment:
I'm not sure if this eases the case, but alternatively we could use a spy to return expected browser values if they're not attached to the window object.

`spyOn(window, 'performance').andReturn({ 'navigation': { type: '0', redirectCount: '0' }, 'timing': { 'domComplete: '123456' } })`

Reply:
I'm not sure I understand the difference between these two options so I'm happy to bow to your expertise on this. Is your change worth doing for a technical reason (does this create a window.performance object outside the scope of the tests that could cause problems?) or is it just neater?

Reply:
Nope, I guess it's just the `jasmine` way of doing it.